x-pack/plugins/observability/public/application/index.tsx

```diff
@@ -128,6 +128,8 @@ export const renderApp = ({
     element
   );
   return () => {
+    // This needs to be present to fix https://github.com/elastic/kibana/issues/155704
```

Comment:
```suggestion
    // This needs to be present to fix https://github.com/elastic/kibana/issues/155704
    // Since overview page has lens embeddable integrated via exploratory view which is using sessions. 
    // So this needs to be called when moving away from the app.
```